### PR TITLE
Feat/services section

### PR DESCRIPTION
**Pull request by ermias5**

### CODE REVIEW

File: components/services/service-card.tsx
Comment (Melka1) [inline components/services/service-card.tsx:1]: This is not a card, it is a list of card, bring the wrapping div here, and change the name of the component.

File: components/footer-section/footer.tsx
Comment (Melka1) [inline components/footer-section/footer.tsx:1]: check the default values of you use, for example, flex, default value flex-direction:row, line-14, check some other places to remove code that does not impact the DOM.
line 20, i don't get the point of wrapping the a in div just to put one component inside.

File: components/contact-section/contact.tsx
Comment (Melka1) [inline components/contact-section/contact.tsx:1]: line 35:  if the ' sign is creating a problem, just use entity code, &apos; for apostrophe, remove the curly braces, it becomes,  ">Let&apos;s Design Together<",  check out https://www.w3schools.com/html/html_entities.asp

File: components/aboutMe/AboutMe.tsx
Comment (Melka1) [inline components/aboutMe/AboutMe.tsx:1]: by default next-image is lazy loaded, please check the documentation

**Review by Melka1 — CHANGES_REQUESTED** [review]
I haven't finished it make the changes and get back to me fast.

**Comment by ermias5** [comment]
update branches and open new latest pr

**Pull request closed without merging**